.flake8

```diff
@@ -35,10 +35,8 @@ per-file-ignores =
   # https://github.com/PyCQA/flake8/issues/1079
   #     F811 redefinition of unused '...'
   stubs/*.pyi: E301, E302, E305, E501, E701, E741, F401, F403, F405, F822, Y026, Y027
-  stdlib/@python2/*.pyi: E301, E302, E305, E501, E701, E741, F401, F403, F405, F822, Y026, Y027
-  stdlib/@python2/typing.pyi: E301, E302, E305, E501, E701, E741, F401, F403, F405, F811, F822, Y026, Y027
   stdlib/typing.pyi: E301, E302, E305, E501, E701, E741, F401, F403, F405, F811, F822, Y026, Y034
 
 # We are checking with Python 3 but many of the stubs are Python 2 stubs.
 builtins = buffer,file,long,raw_input,unicode,xrange
```

Comment:
We can probably get rid of this line too

Reply:
Oh, good point.

Reply:
#7570